### PR TITLE
Ground truth roomba turning

### PR DESCRIPTION
**Pull request by asaba96**
Ground truth roomba turning. Depends on a PR in msgs. 

I created a new topic for the new messages, as I have not had a chance to transition everything over to the new message type.

### CODE REVIEW

File: scripts/roomba.py [hunk 1]
```diff
@@ -106,6 +106,10 @@ def __init__(self, namespace):
                                           Twist,
                                           queue_size=10)
 
+        self._state_publisher = rospy.Publisher('{}state'.format(namespace),
```
Comment (amiller27) [inline scripts/roomba.py]: This topic needs a better name, like not_moving or something

File: src/simulator_adapter.py [hunk 2]
```diff
@@ -375,13 +399,22 @@ def obstacle_odom_callback(msg, topic, data={}):
             Float64,
             queue_size=0)
 
+    global roomba_states
```
Comment (amiller27) [inline src/simulator_adapter.py]: Don't need global here, these are all global

File: scripts/roomba.py (hunk 1 above)
Comment (asaba96) [reply scripts/roomba.py]: `turning` okay?

Comment (asaba96) [reply scripts/roomba.py]: changed to turning

File: src/simulator_adapter.py (hunk 2 above)
Comment (asaba96) [reply src/simulator_adapter.py]: fixed

**Review by amiller27 — APPROVED** [review]

**Pull request merged**